content/docs/integrations/iframe/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: IFrame Plugin
+publishedDate: '2022-04-19T10:00:00.0Z'
+description: How to enable the IFrame plugin
+
+humanName: IFrame Plugin
+logoImage: 'iframe.png'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+This is a plugin that renders an Iframe card that can be added to your entity page
```

Comment:
Can it be added to the homepage too? Seems like it might be useful there.

You might need to clarify what an entity page is. That's kinda internal terminology.

Could you do a quick grammar check please? I spot a missing full stop in the very first sentence. 😄

Reply:
> Can it be added to the homepage too?

Not quite yet, but I'll make sure to add it as part of this work.

I'll do another pass.